pages/general-information-and-resources/washington-dc.md

```diff
@@ -35,7 +35,7 @@ cSpell: ignore capitalweather,Thurgood,Wich,Greenberry,Totten,Smar,WMATA
           <strong>Office / Seats</strong>
         </td>
         <td class="col-value">
-           TTS is located on the 6th floor, 4th wing (6400 rooms, in the infill). It is shared with other FAS colleagues (hi friends!).
+           As of 4/4/25, TTS's location in the building is TBD. <a href="https://insite.gsa.gov/employee-resources/return-to-office-information/1800-f-restacking-plans">See Insite</a> for the latest maps and look for the space(s) listed for FAS.
```

Comment:
TTS's isnt in cspell, can be fixed by putting in front matter 
---
cSpell: ignore TTS's
---

But suggest fixing to just `TTS` to fix